webrtc.html

```diff
@@ -4894,6 +4894,7 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
              RTCRtcpParameters                         rtcp;
              sequence&lt;RTCRtpCodecParameters&gt;           codecs;
              RTCDegradationPreference                  degradationPreference = "balanced";
+             boolean                                   upscaleAllowed = false;
```

Comment:
The meeting notes reflect that we decided to have the default be true.  Also, there's the whole "don't default to a falsy value" thing, so even we wanted to have not allowed be the default, we'd need to make it an enum.  Perhaps we should just make it an enum now, with a more clear name:

boolean resolutionScalingPolicy  = "up-and-down"  // Other option: "down-only"